public/main.js

```diff
@@ -0,0 +1,199 @@
+let button = document.getElementById('start')
+let status = document.getElementById('status')
+let squares = Array.from(document.getElementsByClassName('square'))
+let playerX = { name: 'X', moves: [], symbol: 'X' };
+let playerO = { name: 'O', moves: [], symbol: 'O' };
```

Comment:
You don't need to fully create the variable names here. You could just initialize them like you did with `currentPlayer` below since you are creating them all over again when you start the game anyway